mbrl/util/replay_buffer.py

```diff
@@ -184,10 +277,18 @@ class ReplayBuffer:
             number of steps. Defaults to ``None`` in which case no trajectory
             information will be kept. The buffer will keep trajectory information
             automatically using the done value when calling :meth:`add`.
+        min_trajectory_length (int, optional): if given, than it is assumed to be the minimal
+            trajectory length of a rollout.
+        sequence_length (int, optional): length of the sequences in batch -> batch size is therefore
+            technically sequence_length * specified batch size.
+        use_last_transition_in_sequence (bool, optional): in some cases the reward of last step is
```

Comment:
Can you give an example of the use case where we don't want to use the last transition? Could this be potentially be caught by the `done` status?

Reply:
This is not required for PDDM. I had a project in mind where it was easier to learn on batches where the last reward was discarded (was in model-free RL for Q-learning). We can remove this, the (1-done) trick works well too.